config/initializers/ab_tests.rb

```diff
@@ -97,11 +97,11 @@ def self.all
     },
   ).freeze
 
-  SOCURE_IDV_SHADOW_MODE = AbTest.new(
+  SOCURE_IDV_SHADOW_MODE_FOR_NON_DOCV_USERS = AbTest.new(
```

Comment:
Just an idle musing from someone who hasn't looked at IDP code since last year: I think if I didn't have all the context of this PR in mind, I'd expect this to have a `SOCURE_IDV_SHADOW_MODE_FOR_DOCV_USERS` corollary. This isn't particularly sensible, of course.